src/backends/pyplot.jl

```diff
@@ -176,15 +176,7 @@ function add_pyfixedformatter(cbar, vals::AVec)
 end
 
 function labelfunc(scale::Symbol, backend::PyPlotBackend)
-    if scale == :log10
-        x -> PyPlot.LaTeXStrings.latexstring("10^{$x}")
-    elseif scale == :log2
-        x -> PyPlot.LaTeXStrings.latexstring("2^{$x}")
-    elseif scale == :ln
-        x -> PyPlot.LaTeXStrings.latexstring("e^{$x}")
-    else
-        x -> PyPlot.LaTeXStrings.latexstring(convert_sci_unicode(x))
```

Comment:
I wonder if this else statement is still triggered?

Reply:
It is pretty important as it fixed `Serif` fonts for pyplot

Reply:
It's triggered as the default value in `labelfunc_tex`